src/app/pages/ArticlePage/PagePromoSections/TopStoriesSection/index.jsx

```diff
@@ -81,7 +80,6 @@ const TopStoriesSection = ({ content }) => {
       <StyledSectionLabel
         labelId={LABEL_ID}
         columnType="secondary"
-        backgroundColor={C_GREY_2}
         script={script}
         service={service}
       >
```

Comment:
Here I have removed the backgroundColor prop and instead added a background-color delcaration to the StyledSectionLabel component. I did this to save me using useTheme() or a direct import. Is this the best thing to do here?

Reply:
That gets passed through (eventually!) to the psammead-section-label which has a default prop of:
```
LinkTitle.defaultProps = {
  backgroundColor: C_GHOST,
};
```
So it _may_ be trying to override that style if you declare it in the higher `StyledSectionLabel` component. Thought there would be snapshot change around that though 🤔

Reply:
Ah yes, I've just realised this was overwritten anyway by the default value. I was probably relying on the snapshot too much to point this out! I will update.